### ISSUE TITLE
Response header support

### ISSUE THREAD

**Issue by nbfritz** [mrin9/RapiDoc#25]
_First off, this is a fantastic viewer. Thank you!_

I noticed that when I add headers to a given response object, that data is not included in the output.

I wanted to try to submit a PR for you to save you some time, but I really don't know how you would want to display the list of headers...

It does look like you collected all of the information into a variable at https://github.com/mrin9/RapiDoc/blob/master/src/components/api-response.js#L151 but there's no code to actually use the `headersForEachRespStatus`. Is this particular thing still a work in progress?

Here's the header code I was using -- it's parsing correctly into that variable as far as I can tell...

```
      responses:
        201:
          description: "Successfully created with location header set for new user"
          headers:
            Location:
              description: "URL for the newly created user"
              schema:
                type: string
                example: "http://localhost/api/user/1"
 ```

**Comment by nbfritz** [comment]
FWIW, here's where I'm using it:
* https://shrtnr.github.io/api-docs/
* https://github.com/shrtnr/api-docs

**Comment by mrin9** [comment]
I do have a plan to show them. but no work is in progress and I will be very happy to receive a PR.
The way I envision it is, another tab for them besides 'EXAMPLE' and 'MODEL' ( just like the way you see them after you "TRY" an api
![response headers](https://user-images.githubusercontent.com/2042338/57300465-e165bf80-70a4-11e9-8e90-213c3566bca4.png)

**Comment by mrin9** [comment]
This should now be fixed in 5.0.1 release.

> I added a separate section for response headers instead of creating a tab as  I mentioned earlier. This is due to the `response header` can have their own schema & example where as the tabs are  dedicated to `response` schema and example

**Comment by nbfritz** [comment]
Because I'm pulling your CDN copy it worked immediately. Looks great! Thank
you!
